src/core/schema/src/main/scala/ImageSchema.scala

```diff
@@ -35,9 +34,18 @@ object ImageSchema {
   def isImage(df: DataFrame, column: String): Boolean =
     df.schema(column).dataType == columnSchema
 
-  private[spark] def loadLibraryForAllPartitions[T:ClassTag](rdd: RDD[T], lib: String):RDD[T] = {
-    def perPartition(it: Iterator[T]):Iterator[T] = {
-      new NativeLoader("/nu/pattern/opencv").loadLibraryByName(lib); it }
-    rdd.mapPartitions(perPartition, preservesPartitioning = true)
+  /**
```

Comment:
@mhamilton723 This is a private thing, so shouldn't it be a simple comment rather than a documentation one?  (Tell me if so and I'll fix it.)